plat/xen/x86/link64.lds.S

```diff
@@ -78,25 +78,8 @@ SECTIONS
 
 	TLS_SECTIONS
 
-	/* Data */
-	_data = .;
-	.data : {
-		*(.data)
-		*(.data.*)
-	}
-	_edata = .; /* End of data included in image */
-
-	/*
-	 * NOTE: linker will insert any extra sections here,
-	 * just before .bss
-	 */
+	DATA_SECTIONS
 
-	.bss : {
-		__bss_start = .;
-		*(.bss)
-		*(.bss.*)
-	}
-	. = ALIGN(__PAGE_SIZE);
 	_end = . ;
```

Comment:
For consistency.

```suggestion
	_end = .;
```

Reply:
:+1: That's fine, I tried to touch as few lines as possible but I agree, it is worth fixing this.